src/async-storage.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { EppoAsyncStorage } from './async-storage';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+describe('EppoAsyncStorage', () => {
+  const storageKey = '@eppo/sdk-cache-ufc';
```

Comment:
With Android, we encountered users encountering issues changing up API keys on devices. I wonder if we should also make react native storage isolated per API key.

Reply:
Also, should this just be `STORAGE_KEY`?